src/cli.ts

```diff
@@ -65,8 +65,8 @@ export class Cli implements ICli {
     }
 
     private async getToolLocation(cmd: string): Promise<string> {
-        let toolLocation = path.resolve(Platform.getUserHomePath(), '.vs-openshift', this.tools[cmd].cmdFileName);
-        const toolDlLocation = path.resolve(Platform.getUserHomePath(), '.vs-openshift', this.tools[cmd].dlFileName);
+        let toolLocation = path.resolve(Platform.getUserHomePath(), `.vs-openshift\\cache\\${this.tools[cmd].name}\\${this.tools[cmd].version}`, this.tools[cmd].cmdFileName);
```

Comment:
[path.resolve(...paths)](https://nodejs.org/api/path.html#path_path_resolve_paths) allows you to array of path' segments and avoid to deal with path separators directly. Instead of using strings with `\\` inside, which would work only for windows, just pass each segment as a parameter.

``` JavaScript
path.resolve(Platform.getUserHomePath(), '.vs-openshift', 'cache', cmd, this.tools[cmd].version)
```

Reply:
Would be also good to have a template method for generating the path to avoid code duplication.